python/hail/methods/statgen.py

```diff
@@ -13,41 +13,51 @@
            root=strlike,
            block_size=integral)
 def linreg(dataset, ys, x, covariates=[], root='linreg', block_size=16):
-    """Test each variant for association with multiple phenotypes using linear regression.
-
-    .. warning::
-
-        :py:meth:`.linreg` uses the same set of samples for each phenotype,
-        namely the set of samples for which **all** phenotypes and covariates are defined.
-
-    **Annotations**
-
-    With the default root, the following four variant annotations are added.
-    The indexing of the array annotations corresponds to that of ``y``.
-
-    - **va.linreg.nCompleteSamples** (*Int*) -- number of samples used
-    - **va.linreg.AC** (*Double*) -- sum of input values ``x``
-    - **va.linreg.ytx** (*Array[Double]*) -- array of dot products of each response vector ``y`` with the input vector ``x``
-    - **va.linreg.beta** (*Array[Double]*) -- array of fit effect coefficients, :math:`\hat\beta_1`
-    - **va.linreg.se** (*Array[Double]*) -- array of estimated standard errors, :math:`\widehat{\mathrm{se}}`
-    - **va.linreg.tstat** (*Array[Double]*) -- array of :math:`t`-statistics, equal to :math:`\hat\beta_1 / \widehat{\mathrm{se}}`
-    - **va.linreg.pval** (*Array[Double]*) -- array of :math:`p`-values
-
-    :param ys: list of one or more response expressions.
-    :type ys: list of str
-
-    :param str x: expression for input variable
-
-    :param covariates: list of covariate expressions.
-    :type covariates: list of str
-
-    :param str root: Variant annotation path to store result of linear regression.
-
-    :param int variant_block_size: Number of variant regressions to perform simultaneously.  Larger block size requires more memmory.
-
-    :return: Variant dataset with linear regression variant annotations.
-    :rtype: :py:class:`.VariantDataset`
-
+    """Test each row for association with response variables using linear regression.
+
+    Examples
+    --------
+
+    >>> dataset_result = methods.linreg(dataset, [dataset.pheno.height], dataset.GT.num_alt_alleles(),
+    ...                                 covariates=[dataset.pheno.age, dataset.pheno.isFemale])
+
+    Warning
+    -------
+    :meth:`linreg` uses the same set of columns for each phenotype, the set of
+    columns for which **all** response variables and covariates are defined.
+
+    Notes
+    -----
+
```

Comment:
We lost the model so it's very confusing what is actually being returned (e.g. what is `\hat\beta_1`?). How about I make a PR adding the context back in once this PR is merged?

Reply:
yeah, sounds good.